include/seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm_banded.hpp

```diff
@@ -101,9 +101,26 @@ class pairwise_alignment_algorithm_banded :
     {
         using std::get;
 
+        thread_local score_matrix_single_column<int32_t> alignment_matrix{};
+        coordinate_matrix<uint32_t> index_matrix{};
+        size_t const band_size = upper_diagonal - lower_diagonal + 1;
```

Comment:
```suggestion
        assert(upper_diagonal >= lower_diagonal);
        size_t const band_size = upper_diagonal - lower_diagonal + 1;
```

Reply:
this is in the final version already fixed.